oid4vc-manager/tests/oid4vp/implicit.rs

```
@@ -171,23 +143,46 @@ async fn test_implicit_flow() {
     .unwrap();
 
     // Create a verifiable presentation using the JWT.
-    let verifiable_presentation =
+    let verifiable_presentation_jwt =
         Presentation::builder(subject_did.parse().unwrap(), identity_core::common::Object::new())
             .credential(Jwt::from(jwt))
             .build()
             .unwrap();
 
-    let verifiable_presentation_input = PresentationInputType::Presentation(Box::new(verifiable_presentation));
+    let verifiable_presentation_jwt = VerifiablePresentationJwt::builder()
+        .iss(subject_did.clone())
+        .sub(subject_did)
+        .aud("my-client-id".to_string())
+        .nonce("nonce".to_string())
+        // TODO: make this configurable.
+        .exp((Utc::now() + Duration::minutes(10)).timestamp())
+        .iat((Utc::now()).timestamp())
+        .verifiable_presentation(verifiable_presentation_jwt)
+        .build()
+        .unwrap();
 
-    // Generate the authorization_response. It will include both an IdToken and a VpToken.
-    let authorization_response: AuthorizationResponse<OID4VP> = provider_manager
-        .generate_response(
-            &authorization_request,
-            AuthorizationResponseInput {
-                verifiable_presentation_input,
-                presentation_submission,
-            },
+    let verifiable_presentation_jwt = jwt::encode(
+        subject.clone(),
+        Header {
+            alg: Algorithm::EdDSA,
+            ..Default::default()
+        },
+        &verifiable_presentation_jwt,
+        "did:key",
```

here too I would suggest ```const TEST_DID_METHOD = "did:key";``` at the top of the test module or fn and replace all hardcoded instances with the const
A bit nitpicky this comment, but hardcoded data is the most error prone thing so always nice when we can minimalize that.